modules/openxAnalyticsAdapter.js

```diff
@@ -364,94 +366,93 @@ function onSlotLoaded({ slot }) {
         eventStack[auctionId] = null;
       }
       delete loadedAdSlots[auctionId];
-    }, analyticsConfig.slotLoadWaitTime);
+    }, analyticsConfig.payloadWaitTime);
   }
 }
 
-let openxAdapter = Object.assign(adapter({ urlParam, analyticsType }), {
-  track({ eventType, args }) {
-    if (!checkInitOptions()) {
-      send(eventType, {}, null);
-      return;
-    }
-
-    let info = Object.assign({}, args);
+let openxAdapter = Object.assign(adapter({ urlParam, analyticsType }));
 
-    if (info && info.ad) {
-      info.ad = '';
-    }
+openxAdapter.originEnableAnalytics = openxAdapter.enableAnalytics;
```

Comment:
What is originEnableAnalytics? Did we mean `original`?

Reply:
This follows a standard set by AppNexus's analytics module.